### PR TITLE
Add apib discription

### PR DESCRIPTION
**Pull request by shimastripe**
# WHAT

update Readme.md
- add apiblueprint section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,32 @@ You MUST run this command at the directory which was generated by `new` command.
 $ apig gen
 ```
 
+### API Document
+
+API Documents are auto-generated in `docs/` directory in the form of APIBlueprint.
```
Comment (dtan4) [inline README.md]: - `auto-generated` -> `generated automatically`
- `APIBlueprint` -> [API Blueprint](https://apiblueprint.org/) (with link)

File: README.md [hunk 2]
```diff
@@ -121,6 +121,32 @@ You MUST run this command at the directory which was generated by `new` command.
...
+
+```:docs/
+- index.apib
+- user.apib
+- email.apib
```
Comment (dtan4) [inline README.md]: Format of `tree` command output would be better.

```
docs
├── email.apib
├── index.apib
└── user.apib
```

**Comment by shimastripe** [comment]
@dtan4 Thx! Fixed.

File: README.md [hunk 3]
```diff
@@ -121,6 +121,33 @@ You MUST run this command at the directory which was generated by `new` command.
...
+├── email.apib
+├── index.apib
+└── user.apib
+```
+
+```
+$ npm install -g aglio
+```
+
+[Aglio](https://github.com/danielgtaylor/aglio) is an API Blueprint renderer.
```
Comment (dtan4) [inline README.md]: This sentence should be moved above `npm install -g aglio`. 

```
Aglio is an API Blueprint renderer. Aglio can be installed by
```

**Comment by shimastripe** [comment]
@dtan4 Fixed!

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**